src/defaults.ts

```diff
@@ -5,7 +5,7 @@ import { StandardTargetOptionsMap } from './babel.target.options'
 export const DEFAULT_BABEL_PRESET_OPTIONS: BabelPresetOptions = {
   modules: false,
   useBuiltIns: 'usage',
-  corejs: 2,
+  corejs: 3,
```

Comment:
https://github.com/zloirock/core-js#babelpreset-env mentions in the warning that one should specify both major-minor version, otherwise feature added in minor versions are not injected. (TBH thats an odd behaviour 😩 )

Reply:
Oh, that's pretty new that warning. 
I'll add the latest minor asap